packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -210,10 +214,16 @@ const Layout = () => {
         )}
       </div>
       <Sidebar isOpen={sideBarOpen} onClose={() => setSideBarOpen(false)}>
-        <MenuButton label={t('home')} to="/" tabIndex={sideBarOpen ? 0 : -1} />
-        {menu.map((item) => (
-          <MenuButton key={item.contentId} label={item.label} to={playlistURL(item.contentId)} tabIndex={sideBarOpen ? 0 : -1} />
-        ))}
+        <ul>
+          <li>
+            <MenuButton label={t('home')} to="/" tabIndex={sideBarOpen ? 0 : -1} />
```

Comment:
I know this code is not introduced by you, but are these dynamic tabIndexes needed? The whole Sidebar gets `inert` when `isOpen === false`.

Reply:
I believe I kept this there when implementing `inert` for the sidebar, because you could come in to a state were you sidebar was closed and you were still able to focus the hidden buttons. Potentially due a screen reader bug.

But theoretically that shouldn't occur. So I reevaluated it, tested it again on iOS/Android and it seems like it work. So I committed my change. In the worst case, we can always revert it if this issue reappears during testing.